### PR TITLE
Recently Closed Tabs Fixed

### PR DESCRIPTION
**Pull request by sarinali**
# Description

Had some **really** bad git issues in my previous branch (https://github.com/mozilla/fx-desktop-qa-automation/pull/56) so I just made a new one and this has all the same changes as the old one.

## Bugzilla bug ID

**ID:** 1895900
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1895900

## Type of change

Please delete options that are not relevant.

-  [x] New Test

# How does this resolve / make progress on that bug?

finishes and fixes

# Screenshots / Explanations

n/a

# Comments / Concerns

n/a

### CODE REVIEW

File: modules/util.py [hunk 1]
```diff
@@ -343,6 +343,8 @@ class PomUtils:
         The instance of WebDriver under test.
     """
 
+    allowed_selectors = set([By.ID, By.CLASS_NAME])
```
Comment (ben-c-at-moz) [inline modules/util.py:346]: Since you pulled this out of the method, the name is no longer specific enough to describe what it does.

Comment (ben-c-at-moz) [reply modules/util.py:346]: Also, since you left TAG_NAME in the method, it should be in this set too.

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
Minor change, but otherwise looks good

File: modules/util.py (hunk 1 above)
Comment (sarinali) [reply modules/util.py]: done! fixes are pushed

**Pull request merged**